analysis_options.yaml

```diff
@@ -1,3 +1,4 @@
+include: package:workiva_analysis_options/v2.yaml
 analyzer:
   exclude:
     - example/**
```

Comment:
This is a limitation of the batch that was ran, but can we try removing the specified linter rules below this line?

Ideally we just lean on workiva_analysis_options provides